open-company/open-company-web
# Marketing site refresh

Update some homepage screenshots and show the testimonial section.

Abastract: https://share.goabstract.com/46973a1f-1432-4363-9348-3685b768a2ea

To test:
- check the homepage
- [x] do you see the testimonial section? Good
- [x] copy correspond to abstract? Good
- [x] links on the logos are good? Good
- now logout and go to /login
- dismiss login
- [x] check again if homepage testimonials is ok
- [x] check if testimonials section is ok in pricing
- [x] check if testimonials section is ok on mobile
- merge
- ping @bago2k4 since i need to work the carrot.io branch and deploy there
